language/move-analyzer/editors/code/tests/index.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) The Diem Core Contributors
```

Comment:
ok, you resurrecting the original testing framework first, then I'll add my test cases

Reply:
It's done in #228 which just landed.

Reply:
Great! I probably won't be able to re-review it until later next week, so no great rush!

Reply:
Ok, I happen to have some problems right now. I'm looking for the optimal solution to synchronously waiting for symbol resolution to complete, so PR also needs some time to complete the refactoring.

Reply:
Can't you simply call get_symbols before even the symbolicator thread is created and before the response is sent from the server back to the IDE?

This way the symbolication information is available to the IDE right away (from its perspective) and will continue being available, though it may be a bit outdated until the next file save will trigger recompute.

Reply:
Get_symbols depends on the document content, but the document Symbol PSL protocol does not have this parameter

Reply:
The Document Symbol PSL protocol has only one URI parameter, it seems that I can get the file contents through the RUST file ApI, I will try again，thanks.